src/test/java/it/com/atlassian/tutorial/MyComponentWiredTest.java

```diff
@@ -0,0 +1,25 @@
+package it.com.atlassian.tutorial;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import com.atlassian.plugins.osgi.test.AtlassianPluginsTestRunner;
+import com.atlassian.tutorial.api.MyPluginComponent;
+import com.atlassian.sal.api.ApplicationProperties;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(AtlassianPluginsTestRunner.class)
+public class MyComponentWiredTest {
```

Comment:
why the class has a name ``MyComponentWiredTest``?

Reply:
This test is outdated

Reply:
then why is it here?

Reply:
I delete them in lat commit